cime_config/tests.py

```diff
@@ -161,8 +161,9 @@
             "SMS_Ld2.ne30_oECv3.BGCEXP_BCRC_CNPRDCTC_1850.elm-bgcexp",
             "SMS_D_Ld1.T62_oEC60to30v3.DTESTM",
             "SMS_D_Ld1.ne30_r05_oECv3.A_WCYCL1850",
-            "ERS_Ln9_P96x1.ne4pg2_ne4pg2.F-MMFXX.eam-genmmf",
-            "ERS_Ln9_P96x1.ne4pg2_ne4pg2.F-MMF1-RCEMIP.eam-genmmf",
+            "ERS_Ln9_P96x1.ne4pg2_ne4pg2.F-MMF1.eam-crmout",
+            "ERS_Ln9_P96x1.ne4pg2_ne4pg2.F-MMFXX",
+            "ERS_Ln9_P96x1.ne4pg2_ne4pg2.F-MMF1-RCEMIP",
```

Comment:
Adding `ERS_Ln9_P96x1.ne4pg2_ne4pg2.F-MMF1.eam-crmout` appears to be a duplicate here: `ERS_Ln9.ne4pg2_ne4pg2.F-MMF1.eam-crmout` runs in `e3sm_developer`, which is included in `e3sm_integration`, so both tests are running in `e3sm_integration`. Okay to remove it?

Reply:
Yea, good catch. That's my bad. Do we need to make a PR for this? I can do it right away but I'm not sure whether this deserves a PR or not.

Reply:
The duplicate was removed in E3SM-Project/E3SM#4289 .